packages/contentful--create-contentful-app/src/constants.ts

```diff
@@ -12,6 +12,26 @@ export const IGNORED_EXAMPLE_FOLDERS = [
   'hosted-app-action-templates',
   'function-templates',
 ] as const;
-export const EXAMPLES_PATH = 'contentful/apps/examples/';
+interface GithubPathFunction {
+  (version?: string): string;
+}
+
+export const examples_path: GithubPathFunction = function (version = CURRENT_VERSION) {
```

Comment:
not normally a stickler for naming, but is there any reason not to use a standard camel case naming scheme here?